src/migrations/20240830102144-onboarding-events.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+            CREATE TABLE onboarding_events (
+               event VARCHAR(50) NOT NULL,
+               diff INTEGER NOT NULL,
+               project VARCHAR(255),
```

Comment:
Project definition taken from project table. Same as project id. 
This can be null, because project can be deleted and we need the data to stay forever.